### PR TITLE
EMP-364  CRM7 - Complex page => Case Disposal

### PR DESCRIPTION
**Pull request by BNmoj**
Changes

- amended crm7DisplayCongig.json
- created crm7CaseDisposal.njk
- amended customDisplay.njk

### CODE REVIEW

File: server/views/components/customDisplay/crm7/crm7CaseDisposal.njk [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {% if data.category == 'Category 1' %}
+        {% set fieldValue = data.option.type1 %}
+    {% elif  data.category == 'Category 2' %}
+        {% set fieldValue = data.option.type2 %}
+    {% elif data.category == 'Category 3' %}
+        {% set fieldValue = data.option.type3 %}
+    {% else %}
+        {% set fieldValue = '' %}
+    {% endif %}
+        {# Render "Category" subsection #}
```
Comment (muyinatech) [inline server/views/components/customDisplay/crm7/crm7CaseDisposal.njk:21]: Minor - can we add newline after line 20 and align the code with line 20.

File: server/views/components/customDisplay/crm7/crm7CaseDisposal.njk [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        {% set fields = [
+            { label: "Please select relevant option", value: fieldValue}
+        ] %}
+        {{ formSubsection(data.category, fields) }}
+
+{% endmacro %}
```
Comment (muyinatech) [inline server/views/components/customDisplay/crm7/crm7CaseDisposal.njk:27]: Minor - missing newline

Comment (BNmoj) [reply server/views/components/customDisplay/crm7/crm7CaseDisposal.njk]: done

File: server/views/components/customDisplay/crm7/crm7CaseDisposal.njk (hunk 1 above)
Comment (BNmoj) [reply server/views/components/customDisplay/crm7/crm7CaseDisposal.njk]: done

**Pull request merged**